components/mosart/src/riverroute/RtmIO.F90

```diff
@@ -1335,10 +1343,10 @@ subroutine ncd_io_int_var2_nf(varname, data, flag, ncid, readvar, nt)
        else
           start(1) = 1
           start(2) = 1
-          start(3) = 1
+          !start(3) = 1
```

Comment:
How about removing these entries instead of leaving them as comments?

Reply:
Ok, I will remove the commented code here and rebase the branch.